webcomponents/core/src/components/reveal/deckdeckgo-reveal/deckdeckgo-reveal.tsx

```diff
@@ -1,4 +1,19 @@
-import {Component, h, Element, Prop, Method, State, Host} from '@stencil/core';
+import { Component, h, Element, Prop, Method, State, Host, FunctionalComponent } from '@stencil/core';
+
+const arrayOf = (len: number) => new Array(len).fill(0).map((_, i) => i);
```

Comment:
Quick util to create an Array of specified length. Could be extracted to `utils/` if generally useful.